recipes/source_location/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class SourceLocationConan(ConanFile):
+    name = "source_location"
```

Comment:
```suggestion
    name = "rechip-source_location"
```

I would consider renaming this package, because https://en.cppreference.com/w/cpp/utility/source_location

Reply:
That was the idea to provide a working implementation for older compilers, but once you have the standard implementation, the code will deactivate itself and use the standard one. Hence the same name, because it is the same thing (implementation is a mix of Microsoft's STL, GCC's, and clang's implementation of the source_location). But if you @uilianries think that this should be named differently since anybody can make such compatibility header (then it would be confusing and the name prefix is a good idea), let me know and I will rename it.

Reply:
>  the code will deactivate itself and use the standard one
hmm, so I think it's safe enough to keep `source_location` only.

Reply:
There's other implementations with the exact same repo name https://github.com/search?l=C%2B%2B&q=source_location&type=Repositories

Might be worth adding the author so consumer can easily find the one they want